### PR TITLE
Prep amazon.aws 5.5.2

### PR DESCRIPTION
**Pull request by hakbailey**
##### SUMMARY
Prep amazon.aws 5.5.2 release

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

**Review by mandar242 — APPROVED** [review]
LGTM!

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,21 @@ amazon.aws Release Notes
...
+
+Release Summary
+---------------
+
+This release brings few bugfixes.
+
+Bugfixes
+--------
+
+- Fixes changed status to report False when no change has been made. The module had incorrectly always reported a change. (https://github.com/ansible-collections/amazon.aws/pull/1589)
```
Comment (alinabuzachis) [inline CHANGELOG.rst:19]: Just to give a better understanding about the module that is affected by the change, can you please add it at the beggining?
```suggestion
- cloudwatchevent_rule - Fixes changed status to report False when no change has been made. The module had incorrectly always reported a change. (https://github.com/ansible-collections/amazon.aws/pull/1589)
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,21 @@ amazon.aws Release Notes
...
+- Fixes to the stack trace output, where it does not contain spaces between each character. The module had incorrectly always outputted extra spaces between each character. (https://github.com/ansible-collections/amazon.aws/pull/1615)
```
Comment (alinabuzachis) [inline CHANGELOG.rst:20]:
```suggestion
- lambda_execute - Fixes to the stack trace output, where it does not contain spaces between each character. The module had incorrectly always outputted extra spaces between each character. (https://github.com/ansible-collections/amazon.aws/pull/1615)
```

File: changelogs/changelog.yaml [hunk 3]
```diff
@@ -1732,3 +1732,22 @@ releases:
     - release-summary.yml
     - test-reqs.yml
     release_date: '2023-06-07'
+  5.5.2:
+    changes:
+      bugfixes:
+      - Fixes changed status to report False when no change has been made. The module
```
Comment (alinabuzachis) [inline changelogs/changelog.yaml:1738]:
```suggestion
      - cloudwatchevent_rule - Fixes changed status to report False when no change has been made. The module
```

File: changelogs/changelog.yaml [hunk 4]
```diff
@@ -1732,3 +1732,22 @@ releases:
...
+        had incorrectly always reported a change. (https://github.com/ansible-collections/amazon.aws/pull/1589)
+      - Fixes to the stack trace output, where it does not contain spaces between
```
Comment (alinabuzachis) [inline changelogs/changelog.yaml:1740]:
```suggestion
      - lambda_execute - Fixes to the stack trace output, where it does not contain spaces between
```

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**